.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,4 +16,3 @@ This pull request template helps you complete an application to the **Class of 2
 - [ ] I understand I should not tag or add a reviewer to this Pull Request.
 - [ ] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
 - [ ] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.
-
```

Comment:
line 19 you delete the extra

add line 19 as shown above.

If you restore back changes then this files vanish from here.
Leaving just 2 files down below. And you are done.

Reply:
Okay, I just copied the file from the repo and paste it.